models/curvefi/ethereum/curvefi_ethereum_view_pools.sql

```diff
@@ -34,18 +21,18 @@ regular_pools AS (
         version,
         `name`,
         symbol,
-        pool_address AS pool_address,
-        token_address AS token_address,
-        deposit_contract AS deposit_contract,
-        gauge_contract AS gauge_contract,
-        coin0 AS coin0,
-        coin1 AS coin1,
-        coin2 AS coin2,
-        coin3 AS coin3,
-        undercoin0 AS undercoin0,
-        undercoin1 AS undercoin1,
-        undercoin2 AS undercoin2,
-        undercoin3 AS undercoin3
+        lower(pool_address) AS pool_address,
```

Comment:
ahhh the lower() is here :)

Reply:
maybe it's easier to do at base model instead, just one lower() function on the column at the top

Reply:
yeah I think I've lost track of where what goes lol, happy to move it around